app/routes/profile.js

```diff
@@ -6,5 +6,9 @@ const { Route } = Ember;
 export default Route.extend(AuthenticatedRouteMixin, {
   titleToken() {
     return this.l10n.t('Profile');
+  },
+
+  model() {
+    return this.get('authManager').currentUserModel;
```

Comment:
@niranjan94 The `currentUser` is not returning a model and the properties of the model are also not accessible. Please guide. Thanks

![screen shot 2017-07-20 at 12 48 35 pm](https://user-images.githubusercontent.com/12807846/28405157-cb237074-6d49-11e7-89fd-33241f2e901d.png)